src/main/java/bio/terra/service/filedata/flight/ingest/IngestFileAzurePrimaryDataStep.java

```diff
@@ -100,8 +100,9 @@ public StepResult undoStep(FlightContext context) {
             CommonMapKeys.DATASET_STORAGE_ACCOUNT_RESOURCE,
             AzureStorageAccountResource.class);
     String fileName = getLastNameFromPath(fileLoadModel.getSourcePath());
-    if (!azureBlobStorePdao.deleteDataFileById(
-        fileId, fileName, storageAccountResource, userRequest)) {
+    try {
+      azureBlobStorePdao.deleteDataFileById(fileId, fileName, storageAccountResource, userRequest);
+    } catch (Exception ex) {
```

Comment:
Do you know a more specific exception we could use here? I am always a little worried when it is just `catch(Exception`.

Reply:
The specific exception we ran into in the error was an Interrupted exception. But, I want the undo to still pretty broadly pass no matter the exception so that the rest of the cleanup in the flight still occurs. I think we should keep this broad catch exception.